composer.json

```diff
@@ -14,7 +14,7 @@
     "require": {
         "php": "^7.1.3",
         "symfony/framework-bundle": "^4.3 || ^5.0",
-        "gedmo/doctrine-extensions": "^2.3.4"
+        "gedmo/doctrine-extensions": "^2.3.4 || ^3.0.0-beta2@beta"
```

Comment:
@weaverryan please change the constraint to `^2.3.4 || ^3.0.0` as 3.0 is now released.